vault/logical_system_integ_test.go

```diff
@@ -17,54 +17,58 @@ import (
 	log "github.com/mgutz/logxi/v1"
 )
 
-func TestSystemBackend_enableAuth_plugin(t *testing.T) {
-	coreConfig := &vault.CoreConfig{
-		CredentialBackends: map[string]logical.Factory{
-			"plugin": plugin.Factory,
-		},
-	}
+func TestSystemBackend_Plugin_secret(t *testing.T) {
+	cluster, _ := testSystemBackendMock(t, 1, logical.TypeLogical)
+	defer cluster.Cleanup()
```

Comment:
I tried `t.Cores[0].unloadMounts()` in the `Cleanup()` func and it works, but I feel like that is not the proper way to clean up.

This results in a panic:
```
for _, core := range t.Cores {
	if err := core.Shutdown(); err != nil {
		fmt.Printf("Error with core shutdown: %s", err)
	}
}
```